Xamarin.Forms.Platform.iOS/CollectionView/GroupableItemsViewController.cs

```diff
@@ -226,15 +226,15 @@ int GetLayoutSpanCount()
 
 			if (itemsLayout is GridItemsLayout)
 			{
-				nfloat itemSpacing = itemsViewLayout.GetMinimumInteritemSpacingForSection(collectionView, itemsViewLayout, section);
-
 				if (scrollDirection == UICollectionViewScrollDirection.Horizontal)
 				{
-					return new UIEdgeInsets(itemSpacing + uIEdgeInsets.Top, lineSpacing + uIEdgeInsets.Left,
+					nfloat itemSpacing = itemsViewLayout.GetMinimumInteritemSpacingForSection(collectionView, itemsViewLayout, section);
+
+					return new UIEdgeInsets(uIEdgeInsets.Top, itemSpacing + uIEdgeInsets.Left,
 						uIEdgeInsets.Bottom, uIEdgeInsets.Right);
 				}
 
-				return new UIEdgeInsets(lineSpacing + uIEdgeInsets.Top, itemSpacing + uIEdgeInsets.Left,
+				return new UIEdgeInsets(lineSpacing + uIEdgeInsets.Top, uIEdgeInsets.Left,
 					uIEdgeInsets.Bottom, uIEdgeInsets.Right);
 			}
 
```

Comment:
Isn't line 243 also incorrect as it adds line spacing to the left side, instead of the itemSpacing you replaced one of the cases to? The code there is currently:
return new UIEdgeInsets(uIEdgeInsets.Top, lineSpacing + uIEdgeInsets.Left, [...]

Reply:
243 isn't wrong; because we're not dealing with a grid layout there, the inter-item spacing doesn't have any meaning. Unfortunately, at a second look 233 _is_ wrong. With a horizontal scroll direction, the horizontal item spacing maps to GetMinimumLineSpacingForSection, so GetMinimumInteritemSpacingForSection is the wrong value to use.

Reply:
Okay, should be fixed now.

Reply:
Thank you for the elaboration and fix!